src/parcels/_core/basegrid.py

```diff
@@ -20,6 +22,8 @@ class GridType(IntEnum):
 
 
 class BaseGrid(ABC):
+    """Docstring for Basegrid class"""
```

Comment:
This needs to be expanded? Especially since it's the first one on the list of Classes, it looks a bit strange ("meta"?)